config/config.go

```diff
@@ -530,12 +530,13 @@ func DefaultFuzzConnConfig() *FuzzConnConfig {
 
 // MempoolConfig defines the configuration options for the Tendermint mempool
 type MempoolConfig struct {
-	RootDir   string `mapstructure:"home"`
-	Recheck   bool   `mapstructure:"recheck"`
-	Broadcast bool   `mapstructure:"broadcast"`
-	WalPath   string `mapstructure:"wal_dir"`
-	Size      int    `mapstructure:"size"`
-	CacheSize int    `mapstructure:"cache_size"`
+	RootDir          string `mapstructure:"home"`
+	Recheck          bool   `mapstructure:"recheck"`
+	Broadcast        bool   `mapstructure:"broadcast"`
+	WalPath          string `mapstructure:"wal_dir"`
+	MaxTxs           int    `mapstructure:"max_txs"`
```

Comment:
Sorry, I think I take back my original suggestion for now just to air on the side of breaking fewer things per release.

One day we should do a review/cleanup of the config file perhaps, and then we can fix all the name. Might be better than having one off breaking changes in naming that everyone has to fix ...